deploy/charts/openebs-monitoring/dashboards/Jiva/jiva-volumes.json

```diff
@@ -1102,7 +1100,7 @@
         "allValue": null,
         "current": {},
         "datasource": "$datasource",
-        "definition": "label_values(openebs_size_of_volume{openebs_jiva_label=\"$openebs_jiva_label\"}, openebs_pv)",
+        "definition": "label_values(openebs_size_of_volume{openebs_jiva_label=\"$openebs_jiva_label\", openebs_cstor_label=\"\"}, openebs_pv)",
```

Comment:
@Ab-hishek -- does the same thing needs to be done for cstor volume dashboard?

Reply:
Ahh!! Nice catch. Will need to cstor as well.